### PR TITLE
Added support for twf.user scope through OpenID scope

### PR DESCRIPTION
**Pull request by gerkevgarderen**
Updated the OAuthProvider login to allow the twf.user scope by adding OpenID scope if it is not yet included.

### CODE REVIEW

**Comment by gerkevgarderen** [comment]
Based on #122

File: src/Secure/Provider/OAuthProvider.php [hunk 1]
```diff
@@ -44,18 +48,6 @@ class OAuthProvider extends AbstractProvider
      */
     public const SCOPE_ORGANISATION_USER = 'twf.organisationUser';
 
-    /**
-     * This scope is required in order to receive and use refresh tokens.
-     */
-    public const SCOPE_OFFLINE_ACCESS = "offline_access";
```
Comment (willemstuursma) [inline src/Secure/Provider/OAuthProvider.php]: Why did you remove this constant? Is something wrong with it?

**Comment by gerkevgarderen** [comment]
Turns out to be changes in an older version of the file. Checking again.
